examples/large_models/gpt_fast/handler.py

```diff
@@ -44,15 +44,23 @@ def __init__(self):
 
     def initialize(self, ctx):
         self.context = ctx
+        properties = self.context.system_properties
         rank = maybe_init_dist()
 
         self.local_rank = rank if rank is not None else 0
```

Comment:
if rank is none, local rank should be equal to device_id

Reply:
Thanks for the suggestion Li. Are you suggesting that we implement the handler in this way?
```
def initialize(self, ctx):
        self.context = ctx
        properties = self.context.system_properties
        gpu_id = properties.get("gpu_id")
        rank = maybe_init_dist()

        self.local_rank = rank if rank is not None else int(gpu_id)

        if torch.cuda.is_available():
            self.map_location = "cuda"
            self.device = torch.device(self.map_location + ":" + str(self.local_rank))

            torch.cuda.set_device(self.local_rank)
```

Reply:
please add "raise exception" when gpu_id < 0

Reply:
Understood. I will include the exception